docker/README.md

```diff
@@ -25,14 +25,29 @@
 
 5) _Optional:_ run container setup script
     ```sh
-    make setup
+    make setup-default
+
+    # If you prefer to manually configure setup, you can just do make setup and go through the prompts
+    make setup-interactive
     ```
 6) Acces the environment from any terminal window
     ```sh
     docker exec -it mep3-devel bash
     ```
     Graphical applications started inside this terminal will use your existing Xorg session to display.
 
+
+## Code Server
+If you prefer to use browser based VS Code, you can start it in the container and then access it locally through your browser at `localhost:8080`
+    
+```sh
+# This will start the VS code server with your mep3 repo
+make start-code-server
```

Comment:
Da li dodati `xdg-open http://localhost:<port> &` u ovaj target da automatski otvori Code server?

Reply:
To deluje kao dobra ideja, dodatnih bih jos mozda dodao da ne pokrece novu instancu, ako je vec pokrenut code server.